auth/generate-google-id-jwt2.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+import requests
+import time
+
+from pprint import pprint
+
+from google.auth import crypt, jwt
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+# generates a signed JSON Web Token
+
+with open('/usr/local/google/home/ensonic/robco_gob/cloud-robotics/src/bootstrap/robot/robot-credentials.json', 'r') as fh:
+    service_account_info = json.load(fh)
+
+signer = crypt.RSASigner.from_service_account_info(service_account_info)
+
+now = int(time.time())
+payload = {
+    'iat': now,
+    # expires after one hour.
+    'exp': now + 3600,
+    'iss': service_account_info['client_email'],
+    # the URL of the target service.
+    'target_audience': 'https://registry.endpoints.robco-166608.cloud.goog',
+    # Google token endpoints URL, FIXME: use discovery doc
+    'aud': 'https://www.googleapis.com/oauth2/v4/token'
```

Comment:
I don't think this URL should be hardcoded either (se also below). This should probably taken from api discovery.